pkg/make/builder_test.go

```diff
@@ -0,0 +1,180 @@
+package make
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"strconv"
+	"testing"
+
+	"github.com/Masterminds/semver"
+	"github.com/stretchr/testify/require"
+)
+
+func helperCommandContext(ctx context.Context, command string, args ...string) (cmd *exec.Cmd) {
+	cs := []string{"-test.run=TestHelperProcess", "--", command}
+	cs = append(cs, args...)
+	if ctx != nil {
```

Comment:
ctx should never be nil. eliminate the non context branch from here and always pass the ctx param.